roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -25,6 +25,7 @@ jmxexporter_enabled: false
 jmxexporter_install_path: /opt/prometheus/
 jmxexporter_jar_path: /opt/prometheus/jmx_prometheus_javaagent.jar
 kafka_broker_jmxexporter_port: 8080
+zookeeper_jmxexporter_port: 8080
```

Comment:
This all looks great, we ~might~ want to introduce two variables like so 
kafka_broker_jmxexporter_enabled: "{{jmxexporter_enabled}}"
zookeeper_jmxexporter_enabled: "{{jmxexporter_enabled}}"

Do you think theres a chance that customers would want one on and one off? This way they set jmxexporter to true and both get it, but they could also be selective.

Reply:
Good point, I think most customers would have the JMX exporter on all components, but I like the idea in giving them more flexibility in the setup. Will add. Thanks for the review!